src/services/kubernetes/web-deployment.yaml

```diff
@@ -17,3 +17,7 @@ spec:
           image: reactivetradercloud/nginx:${BUILD_VERSION}
           ports:
             - containerPort: 3000
+          resources:
+            requests:
+              memory: "10M"
+              cpu: "15m"
```

Comment:
Why so little memory for the client, is it because nginx doesn't need a lot?

Reply:
In our deployments, we normally register values under 10MB except for some spikes around 15 to 25. I think this one is correct